From 0ee1eff1b970522c1626cc3777adfd9496b809d9 Mon Sep 17 00:00:00 2001
From: Allen Xu <b45815@freescale.com>
Date: Wed, 1 Oct 2014 17:25:49 -0500
Subject: [PATCH] MLK-9659 ARM: dts reduce QSPI freq on imx6sx-17x17-arm2
 board

According to design spec on QSPI IO freq, reduce the freq to 29MHz.

Non-DQS mode:
-          SDR: 60MHz
-          DDR: 30Mhz

Signed-off-by: Allen Xu <b45815@freescale.com>
(cherry picked from commit da5d7175b84db47b8269a531e22a919d10d4bce9)
---
 arch/arm/boot/dts/imx6sx-17x17-arm2.dts | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/arch/arm/boot/dts/imx6sx-17x17-arm2.dts b/arch/arm/boot/dts/imx6sx-17x17-arm2.dts
index 74443c3..971416c 100644
--- a/arch/arm/boot/dts/imx6sx-17x17-arm2.dts
+++ b/arch/arm/boot/dts/imx6sx-17x17-arm2.dts
@@ -444,7 +444,7 @@
 		#address-cells = <1>;
 		#size-cells = <1>;
 		compatible = "micron,n25q256a";
-		spi-max-frequency = <53000000>;
+		spi-max-frequency = <29000000>;
 		reg = <0>;
 	};
 
@@ -452,7 +452,7 @@
 		#address-cells = <1>;
 		#size-cells = <1>;
 		compatible = "micron,n25q256a";
-		spi-max-frequency = <53000000>;
+		spi-max-frequency = <29000000>;
 		reg = <1>;
 	};
 };
-- 
1.8.0

